app/javascript/spec/reconfigure-vm-form/reconfigure-vm-form.spec.js

```diff
@@ -0,0 +1,173 @@
+import React from 'react';
+import toJson from 'enzyme-to-json';
+import fetchMock from 'fetch-mock';
+import { act } from 'react-dom/test-utils';
+import ReconfigureVmForm from '../../components/reconfigure-vm-form/index';
+import {
+  valueFromHelpers, valueFromHelpersTwo, responseDataOne, responseDataThree, responseDataTwo, responseDataFour,
```

Comment:
seems like this variable is not being used.

Reply:
done.